admin/plugins.php

```diff
@@ -0,0 +1,399 @@
+<?php
+/**
+ * Admin settings helper functions.
+ *
+ * @package performance-lab
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Get info about all the standalone plugins and their status.
+ *
+ * @since n.e.x.t
+ *
+ * @return array of plugins info.
+ */
+function perflab_get_standalone_plugins() {
+	$managed_standalone_plugins = array(
+		'webp-uploads',
+		'performant-translations',
+		'dominant-color-images',
+	);
+
+	return $managed_standalone_plugins;
+}
+
+/**
+ * Renders plugin UI for managing standalone plugins within PL Settings screen.
+ *
+ * @since n.e.x.t
+ */
+function perflab_render_plugins_ui() {
+	require_once ABSPATH . 'wp-admin/includes/plugin-install.php';
+	require_once ABSPATH . 'wp-admin/includes/plugin.php';
+
+	$standalone_plugins = perflab_get_standalone_plugins();
+
+	if ( empty( $standalone_plugins ) ) {
+		return;
+	}
+	?>
+	<div class="wrap">
+		<h1><?php esc_html_e( 'Performance Plugins', 'performance-lab' ); ?></h1>
+		<p><?php esc_html_e( 'The following standalone performance plugins are available for installation.', 'performance-lab' ); ?></p>
+		<div class="wrap">
+			<form id="plugin-filter" method="post">
+				<div class="wp-list-table widefat plugin-install wpp-standalone-plugins">
+					<h2 class="screen-reader-text"><?php esc_html_e( 'Plugins list', 'default' ); ?></h2>
+					<div id="the-list">
+						<?php
+						foreach ( $standalone_plugins as $standalone_plugin ) {
+							perflab_render_plugin_card( $standalone_plugin );
+						}
+						?>
+					</div>
+				</div>
+			</form>
+		</div>
+		<div class="clear"></div>
+	</div>
+	<?php
+}
+
+/**
+ * Render individual plugin cards.
+ *
+ * @since n.e.x.t
+ *
+ * @param string $standalone_plugin Plugin slug as passed from get_standalone_plugins().
+ */
+function perflab_render_plugin_card( string $standalone_plugin ) {
+	$plugin = plugins_api(
+		'plugin_information',
+		array(
+			'slug'   => $standalone_plugin,
+			'fields' => array(
+				'short_description' => true,
+				'icons'             => true,
+			),
+		)
+	);
+	if ( is_object( $plugin ) ) {
+		$plugin = (array) $plugin;
+	}
```

Comment:
A recommendation to improve the "API" here for separation of concerns and future compatibility: I think it would be cleaner to remove the `plugins_api()` call from here and instead change this function to expect the API result (i.e. `array $plugin`) as the parameter. That way we decouple the rendering from the request.

Other than just separation of concerns, this has two benefits:
* We can make the API requests for all the plugins before starting rendering the individual cards. I would recommend that, for example in a `foreach` loop right after the `perflab_get_standalone_plugins()` call.
* It would allow us to change the way we make these API requests in the future without having to break backward compatibility with this function here. For example, if WordPress.org supported batch-requesting this information, we would be able to do so without breaking BC.

Reply:
@felixarntz this has been addressed in https://github.com/WordPress/performance/pull/864/commits/f7c0c292ca0176f3eb81ba586c3d72d89a08eed7 thank you!